changelog.md

```diff
@@ -1,7 +1,32 @@
-# 0.13.2
+# 0.14.0
+
+## Breaking Changes
+* `BOHB4HPO` facade has been renamed to `SMAC4MF` facade (#738)
+* Require `scipy` >= 1.7 (#729)
+* Require `emcee` >= 3.0.0 (#723)
+
+## Major Changes
+* Drop support for Python 3.6 (#726)
+* Added Colab to try SMAC in your browser! (#697)
 
 ## Minor Changes
-* Require emcee >= 3.0.0
+* Added gradient boosting example, removed random forest example (#722)
+* `lazy_import` dependency dropped (#741)
+* Replaced `pyDOE` requirement with `scipy` for LHD design (#735)
+* Uses scrambled Sobol Sequence (#733)
+* Moved to Github actions (#715)
+* Improved testing (#720, #723, #739, #743)
+* Added option `save_results_instantly` in scenario object to save results instantly (#728)
+* Changed level of intensification messages to debug (#724)
```

Comment:
This is duplicated with line 28, I would suggest to delete line 28 and preserve this one